drv/user-leds-api/src/lib.rs

```diff
@@ -22,4 +22,25 @@ impl From<u32> for LedError {
     }
 }
 
+#[derive(
+    Copy, Clone, Debug, Eq, PartialEq, zerocopy::AsBytes, FromPrimitive,
+)]
+#[repr(u8)]
+pub enum LedState {
+    Off = 0,
+    On = 1,
+}
+
 include!(concat!(env!("OUT_DIR"), "/client_stub.rs"));
+
+impl UserLeds {
+    pub fn led_set(&self, index: usize, state: bool) -> Result<(), LedError> {
+        self.led_set_state(
+            index,
+            match state {
+                true => LedState::On,
+                false => LedState::Off,
+            },
+        )
+    }
+}
```

Comment:
Of course a much simpler version of this would be simply:
```rust
impl UserLeds {
    pub fn led_set(&self, index: usize, state: bool) -> Result<(), LedError> {
        match state {
            true => self.led_on(index),
            false => self.led_off(index),
        }
    }
}
```
and dispense with all the other changes, perhaps this was what was intended? Seems too simple for the OP to have bothered raising an issue for, though I don't know Hubris well enough to judge what the benefit of the more complex approach is.